src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/maskinporten/TokenResponse.kt

```diff
@@ -29,5 +29,13 @@ data class TokenResponseWrapper(
 ) {
     private val expiresAt = requestedAt + tokenResponse.expiresIn
     fun expiresIn(now: Instant = Instant.now()): Duration = Duration.between(now, expiresAt)
+
+    fun percentageRemaining(now: Instant = Instant.now()): Double {
+        val expiresIn = expiresIn(now)
+        return if (expiresIn.isZero || expiresIn.isNegative)
+            0.0
+        else
+            100.0 / tokenResponse.expiresIn.dividedBy(expiresIn)
```

Comment:
Denne linjen blir litt vanskelig siden `expiresIn` har to betydninger. Kunne kanskje `tokenLifetime` og `timeToExpire` vært bedre navn?

Reply:
endret lokal variabel til timeToExpire